skopt/space/space.py

```diff
@@ -163,35 +130,53 @@ def __init__(self, low, high, prior="uniform"):
               and upper bounds.
             - If `"log-uniform"`, points are sampled uniformly between
               `log10(lower)` and `log10(upper)`.`
+
+        * `transform` [None or "normalize", optional]:
+            If `transform=normalize`, calling `transform` on X scales X to
+            [0, 1]
         """
         self.low = low
         self.high = high
         self.prior = prior
+        self.transform_ = transform
 
-        if prior == "uniform":
-            self._rvs = uniform(self.low, self.high - self.low)
-            self.transformer = _Identity()
-
-        elif prior == "log-uniform":
-            self._rvs = uniform(
-                np.log10(self.low),
-                np.log10(self.high) - np.log10(self.low))
-            self.transformer = _Log10()
-
-        else:
+        if self.transform_ and self.transform_ != "normalize":
             raise ValueError(
-                "Prior should be either 'uniform' or 'log-uniform', "
-                "got '%s'." % self._rvs)
+                "transform should be normalize, got %s" % self.transform_)
+
+        # Define _rvs and transformer spaces.
+        # XXX: The _rvs is for sampling in the transformed space.
+        # The rvs on Dimension calls inverse_transform on the points sampled
+        # using _rvs
+        if self.transform_ == "normalize":
+            self._rvs = uniform(0, 1)
+            if self.prior == "uniform":
+                self.transformer = Pipeline(
+                    [Identity(), Normalize(low, high)])
+            else:
+                self.transformer = Pipeline(
+                    [Log10(), Normalize(np.log10(low), np.log10(high))]
```

Comment:
But how to address this? (https://github.com/scikit-optimize/scikit-optimize/pull/237#issuecomment-251415962)